patches/minecraft/net/minecraft/server/management/PlayerList.java.patch

```diff
@@ -184,7 +184,7 @@
 +    }
 +
 +    @SuppressWarnings("unused")
-+    public void transferEntityToWorld(Entity p_82448_1_, int p_82448_2_, WorldServer p_82448_3_, WorldServer p_82448_4_, net.minecraft.world.Teleporter teleporter)
++    public void transferEntityToWorld(Entity p_82448_1_, int p_82448_2_, WorldServer p_82448_3_, WorldServer p_82448_4_, net.minecraftforge.common.util.ITeleporter teleporter)
```

Comment:
This breaks binary compatibility, even though Teleporter is an instance of ITeleporter, the JVM doesn't care about that when invoking the method, all it cares about is the sig of the method.

Reply:
Ah, right.

I wasn't sure of the exact handling of this. If that's the case, I can add a temporary redirect with the old signature.